lib/std/Build/Step/Compile.zig

```diff
@@ -741,6 +741,8 @@ fn runPkgConfig(compile: *Compile, lib_name: []const u8) !PkgConfigResult {
             try zig_cflags.appendSlice(&[_][]const u8{ "-D", macro });
         } else if (mem.startsWith(u8, arg, "-D")) {
             try zig_cflags.append(arg);
+        } else if (mem.startsWith(u8, arg, "-Wl,-rpath,")) {
+            try zig_cflags.appendSlice(&[_][]const u8{ "-rpath", arg[11..] });
```

Comment:
imo `"-Wl,-rpath,".len` would read better than a magic number 11.
Even better would be `const wl_rpath_arg_prefix = "-Wl,-rpath,";` and using that constant in both places.

Reply:
Yep, you're right. I've also cleaned up the stray commit from my previous PR